docs/Developers/Rankings.rst

```diff
@@ -0,0 +1,103 @@
+Rankings
+========
+
+The coala community is split into different ranks which have different rights
+and obligations. This page will explain which the ranks are and what you need
+to do to get promoted to a higher rank.
+
+Currently the ranks can be seen `here <https://github.com/orgs/coala/teams>`__
+and are as it follows:
+
+ - newcomers
+ - developers
+ - maintainers
+ - admins
+
+The Newcomer Rank
+-----------------
+
+Everyone starts here. This is the rank where you can get a promotion easiest
+and which has the lowest obligations, and, obviously, the lowest rights.
+
+Rights
+~~~~~~
+
+The newcomers have no write access, and no direct labelling rights. The only
+thing that can be done by them is reviewing by leaving useful comments on
+Pull Requests which help them prove themselves as good reviewers.
+
+How to get promoted
```

Comment:
big heading is ====
ranks are ---------
promotions and ranks are ~~~~

isnt this sane?